lib/tasks/rake_arguments.rb

```diff
@@ -0,0 +1,9 @@
+class RakeArguments
+  def self.task_options
+    ARGV.drop_while { |e| e != '--' }.drop(1).join(' ')
+  end
+
+  def self.more_tasks?
+    ARGV.take_while { |e| e != '--' }.drop(1).any?
```

Comment:
The resulting method could probably be called `command_options` which makes `task_options` make more sense as a name.